Utilities/CodeGeneration/EdFi.Ods.CodeGen/Generators/EntityOrmMappingsForViews.cs

```diff
@@ -18,10 +18,10 @@ namespace EdFi.Ods.CodeGen.Generators
     public class EntityOrmMappingsForViews : GeneratorBase
     {
         private const string NotRendered = null;
-        private readonly IViewsProvider _viewsProvider;
+        private readonly IAuthorizationDatabaseTableViewsProvider _viewsProvider;
         private readonly IDatabaseTypeTranslator _databaseTypeTranslator;
 
-        public EntityOrmMappingsForViews(IViewsProvider viewsProvider, IDatabaseTypeTranslator databaseTypeTranslator)
+        public EntityOrmMappingsForViews(IAuthorizationDatabaseTableViewsProvider viewsProvider, IDatabaseTypeTranslator databaseTypeTranslator)
```

Comment:
Same here, rename the field and constructor param.

Reply:
I fixed it .Thanks